cityofaustin/joplin
# 3388 - Make Joplin pages know their Janis URLs

# Description

What this does:
* Adds a Janis URL table
* Updates the Janis URL table for each page on publish

That's about it.



https://github.com/cityofaustin/techstack/issues/3388

# Types of changes



# How can this be tested?
https://joplin-pr-3388-urls.herokuapp.com/admin/
I related the Janis URLs to each page, so on save you can see "URLs" in the author interface update

# Checklist:
